io/disk/rawread.py

```diff
@@ -55,6 +55,10 @@ def setUp(self):
                                    os.path.basename(
                                        tarball.split('.tar')[0]))
         os.chdir(self.source)
+        d_distro = distro.detect()
+        arch = d_distro.arch
+        if arch == 'x86_64':
+            build.make(self.source, extra_args="clean")
```

Comment:
this hold good on all architecture as it not harm any thing , please move this from condition

Reply:
Sure

Reply:
Updated according to the reviewers comments, Please review again.